src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.cpp

```diff
@@ -209,10 +209,10 @@ real64 HydrofractureSolver< POROMECHANICS_SOLVER >::fullyCoupledSolverStep( real
     }
 
     // currently the only method is implicit time integration
-    dtReturn = nonlinearImplicitStep( time_n, dt, cycleNumber, domain );
+    dtReturn = nonlinearImplicitStep( time_n, dtReturn, cycleNumber, domain );
```

Comment:
Simply change dt to dtReturn here may still have issues, as dt used in different fracture propagation iterations will still be different if timestep cut happens in some of the fracture propagation iteration.